lib/sequence.js

```diff
@@ -272,13 +278,12 @@ Sequence.prototype._setNextCounter = function(doc, callback) {
     };
 
     async.retry(0, retriable.bind(this), callback);
-
 };
 
-module.exports = function(goose){
-    if(arguments.length === 2) {
+module.exports = function (goose) {
+    if (!goose) {
```

Comment:
The previous check was correct. If they pass two arguments, because it's using this function as the plugin, goose is defined and the check pass, wrongly

Reply:
my bad. I've made a test to cover the missing lines from this function.